htdocs/reception/class/reception.class.php

```diff
@@ -1116,7 +1116,7 @@ public function delete(User $user)
 		$this->db->begin();
 
 		// Stock control
-		if (isModEnabled('stock') && !getDolGlobalInt('STOCK_CALCULATE_ON_RECEPTION') && $this->statut > 0) {
+		if (isModEnabled('stock') && getDolGlobalInt('STOCK_CALCULATE_ON_RECEPTION') && $this->statut > 0) {
```

Comment:
I think the test should be more complete. When do you think of
```
if (isModEnabled('stock') && ((getDolGlobalInt('STOCK_CALCULATE_ON_RECEPTION') && $this->statut > 0)
|| (getDolGlobalInt('STOCK_CALCULATE_ON_RECEPTION_CLOSE') && $this->statut == $this::STATUS_CLOSED))) {
```

Reply:
Exact ! :) :+1: